amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -256,18 +331,16 @@ impl Transform {
     /// We can exploit the extra information we have to perform this inverse faster than `O(n^3)`.
     pub fn view_matrix(&self) -> Matrix4<f32> {
         // todo
```

Comment:
nice todo :thinking: 
I know you didn't add it, but if you know how to complete it easily please do ;)

Reply:
I honestly have no idea what this todo wants. I assume it's talking about exploiting that additional information but that's not something I have the knowledge to do.

Reply:
This needs to be tested but I believe this `todo` means something like the following which should be faster than plain matrix inversion:
```rust
let inv_scale = Vector3::new(1.0 / self.scale.x, 1.0 / self.scale.y, 1.0 / self.scale.z);
self.iso.inverse().to_homogeneous().append_nonuniform_scaling(&inv_scale);
```